policies/policy_service.go

```diff
@@ -359,5 +359,23 @@ func (s policiesService) InactivateDatasetByIDInternal(ctx context.Context, owne
 		return errors.Wrap(ErrInactivateDataset, err)
 	}
 
+	return nil
+}
+
+func (s policiesService) DeleteAgentGroupFromAllDatasets(ctx context.Context, aGroupID string, token string) error {
+	ownerID, err := s.identify(token)
+	if err != nil {
+		return err
+	}
```

Comment:
In this layer we usually don't have a log? Is it ensured that someone will log this?